src/kvstore/Part.cpp

```diff
@@ -420,11 +421,55 @@ bool Part::preProcessLog(LogID logId, TermID termId, ClusterID clusterId, const
 }
 
 void Part::cleanup() {
-  LOG(INFO) << idStr_ << "Clean rocksdb commit key";
-  auto res = engine_->remove(NebulaKeyUtils::systemCommitKey(partId_));
-  if (res != nebula::cpp2::ErrorCode::SUCCEEDED) {
-    LOG(WARNING) << idStr_ << "Remove the committedLogId failed, error "
-                 << static_cast<int32_t>(res);
+  LOG(INFO) << idStr_ << "Clean rocksdb part data";
+  // Remove the vertex, edge, index, systemCommitKey data under the part
+  // TODO(pandasheep) Maybe there is a better way.
+  std::vector<std::string> toDelete;
+  std::unique_ptr<KVIterator> vertexIter;
+  std::unique_ptr<KVIterator> edgeIter;
+  std::unique_ptr<KVIterator> indexIter;
+  auto vertexPre = NebulaKeyUtils::vertexPrefix(partId_);
+  auto ret = engine_->prefix(vertexPre, &vertexIter);
+  if (ret != nebula::cpp2::ErrorCode::SUCCEEDED) {
+    LOG(ERROR) << idStr_ << "Remove the part vertex data failed, error "
+               << static_cast<int32_t>(ret);
+    return;
+  }
+  while (vertexIter && vertexIter->valid()) {
+    toDelete.emplace_back(vertexIter->key().str());
+    vertexIter->next();
+  }
+
```

Comment:
use delete range may be a better way?
https://github.com/facebook/rocksdb/wiki/Delete-A-Range-Of-Keys

Reply:
Yeah, `DeleteRange `should be used here. I discussed it with @critical27 offline.

Reply:
has used `DeleteRange `